### PR TITLE
Accuracy and Hiperparams

### LINKED ISSUE #10: Introduce Batch Training
default size of batch 128

### LINKED ISSUE #11: Accuracy and F1 score
Eventualy Area Under Curve

### LINKED ISSUE #13: Wages initialization

### LINKED ISSUE #16: Serialization of database using pickle

### PR DESCRIPTION
**Pull request by mateuszdorobek**
Resolve #10, resolve #13, resolve #16, resolve #11

### CODE REVIEW

File: network.py [hunk 1]
```diff
@@ -23,37 +33,90 @@ def one_hot(y, output_size):
     return t
 
 
-if __name__ == "__main__":
+def check_loss(network, x_check_cost, y_check_cost):
```
Comment (MichalTurski) [inline network.py:36]: Ja bym tutaj nie umieszczał `_check_lost` w nazwie. To nic nie wnosi do rozumienia tego, co się dzieje w funkcji.

File: network.py [hunk 2]
```diff
@@ -23,37 +33,90 @@ def one_hot(y, output_size):
...
+    pickle_off.close()
+    x = mnist.data.astype('float32')
+    y = mnist.target.astype('int64')
+    x /= 255.0
+    X_train, X_test, y_train, y_test = train_test_split(x, y, test_size=0.20, random_state=17)
+    assert (X_train.shape[0] + X_test.shape[0] == mnist.data.shape[0])
+    return X_train[0:1000], X_test, y_train[0:1000], y_test
+
+
+def check_accuracy(network, X_test, y_test):
```
Comment (MichalTurski) [inline network.py:66]: Jak wyżej, moim zdaniem `_test` nie wnosi żadnej informacji.

File: network.py (hunk 1 above)
Comment (mateuszdorobek) [reply network.py:36]: dobra może być x i y

File: network.py (hunk 2 above)
Comment (mateuszdorobek) [reply network.py:66]: ok
